### ISSUE TITLE
add photos to the menu

### ISSUE THREAD

**Issue by Miktau** [Shopify/dawn#3260]
add photos to the menu. Suggest a nice layout or where to have it. 
![Screenshot 2024-02-08 at 14 50 10](https://github.com/Shopify/dawn/assets/61705825/4b150945-2729-4812-84ee-ff83d03d25da)
![Screenshot 2024-02-08 at 14 50 23](https://github.com/Shopify/dawn/assets/61705825/7ac2ba53-7a17-4d97-8b42-8a79c67bbd11)

**Comment by vitalijalbu** [comment]
Yeah would be fine, of course it can be done by code… but better that maintaners do it 😁
